### DISCUSSION TITLE
从 NSObject 的初始化了解 isa · /isa

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#296] [Announcements]
https://draveness.me/isa

**Comment by xiefangzhenz** [comment]
这篇博客看了一天，扩大了视野，还有很多不懂的疑问，过段时间再啃一遍

**Comment by wujunze** [comment]
👍

**Comment by leejunhui** [comment]
16年就有这样的知识深度，灯塔就是灯塔~

**Comment by wanhmr** [comment]
上面打印 binaryWithInteger 方法似乎是反了，打印的应该是左边是低位，右边是高位。

**Comment by Huanhoo** [comment]
"这种行为已经被启用了" 此处有错别字，应该是"这种行为已经被弃用了"

---

*2021-04-02 UPDATES：已修复*

**Comment by sinno93** [comment]
"因为类的指针要按照字节（8 bits）对齐内存，其指针后三位都是没有意义的 0" 这里的8 bits是否应该是8 Bytes? 

---

*2021-04-02 UPDATES：已修复*
